charts/datasaur/Chart.yaml

```diff
@@ -15,13 +15,13 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.5.0
+version: 1.5.1
```

Comment:
Better to be 1.6.0 since there's breaking changes

Reply:
oh yes, i forgot about this. wkt itu gw bilangnya di slack jg 1.6.0

tolong ya @michael-jong-ds

Reply:
updated to 1.6.0 :pray: